ui-admin/src/forms/designer/NewQuestionForm.test.tsx

```diff
@@ -1,31 +1,63 @@
 import React from 'react'
-import { render, screen, fireEvent } from '@testing-library/react'
+import { act, render, screen } from '@testing-library/react'
 import { NewQuestionForm } from './NewQuestionForm'
 import { questionTypeDescriptions } from './questions/questionTypes'
+import userEvent from '@testing-library/user-event'
 
 describe('NewQuestionForm', () => {
   test('renders the default view for a new question', () => {
     //Arrange
     render(<NewQuestionForm onCreate={() => jest.fn()} readOnly={false} />)
 
     //Assert
-    const questionStableIdInput = screen.getByLabelText('Question stable ID')
+    screen.getByLabelText('Question stable ID')
     const questionTypeSelect = screen.getByLabelText('Question type')
-    expect(questionStableIdInput).toBeInTheDocument()
-    expect(questionTypeSelect).toBeInTheDocument()
-    expect((questionTypeSelect as HTMLSelectElement).value).toBe('text')
+    expect(questionTypeSelect).toHaveValue('placeholder')
   })
 
-  test('updates to the appropriate QuestionDesigner when a new question type is selected', () => {
+  test('updates to the appropriate QuestionDesigner when a new question type is selected', async () => {
     //Arrange
-    render(<NewQuestionForm onCreate={() => jest.fn()} readOnly={false} />)
+    render(<NewQuestionForm onCreate={() => jest.fn()} readOnly={false}/>)
 
     //Act
+    const user = userEvent.setup()
     const questionTypeSelect = screen.getByLabelText('Question type')
-    fireEvent.change(questionTypeSelect, { target: { value: 'checkbox' } })
+    await act(() => user.selectOptions(questionTypeSelect, 'checkbox'))
```

Comment:
Generally, `userEvent` methods should handle `act` internally. 
```suggestion
    await user.selectOptions(questionTypeSelect, 'checkbox')
```

Reply:
I added the `act` because I was getting this console error when running the test locally (the test still passed regardless):

```
  console.error
    Warning: An update to NewQuestionForm inside a test was not wrapped in act(...).
    
    When testing, code that causes React state updates should be wrapped into act(...):
    
    act(() => {
      /* fire events that update state */
    });
    /* assert on the output */
    
    This ensures that you're testing the behavior the user would see in the browser. Learn more at https://reactjs.org/link/wrap-tests-with-act
        at NewQuestionForm (/Users/mbemis/Code/broad/pearl/ui-admin/src/forms/designer/NewQuestionForm.tsx:20:11)
```

Reply:
Just out of curiosity, I wonder if this changes if `userEvent.setup` is called before rendering the component.

> We recommend invoking [userEvent.setup()](https://testing-library.com/docs/user-event/setup) before the component is rendered.
https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent

Reply:
I tried it out and I still got the same warning, but I just pushed a commit to at least move the `userEvent.setup` call before the `render`, per the recommendation